cartoframes/data/observatory/catalog/dataset.py

```diff
@@ -98,95 +86,79 @@ def variables_groups(self):
         Returns:
             :py:class:`CatalogList <cartoframes.data.observatory.entity.CatalogList>` List of VariableGroup instances.
 
-        :raises CartoException: If there's a problem when connecting to the catalog.
+        Raises:
+            Exception: if there's a problem when connecting to the catalog.
+
         """
         return get_variable_group_repo().get_all({DATASET_FILTER: self.id})
 
     @property
     def name(self):
         """Name of this dataset."""
-
         return self.data['name']
 
     @property
     def description(self):
         """Description of this dataset."""
-
         return self.data['description']
 
     @property
     def provider(self):
-        """Id of the :obj:`Provider` of this dataset.
-        """
-
+        """Id of the :obj:`Provider` of this dataset."""
         return self.data['provider_id']
 
     @property
     def provider_name(self):
         """Name of the :obj:`Provider` of this dataset."""
-
         return self.data['provider_name']
 
     @property
     def category(self):
-        """Get the :obj:`Category` ID assigned to this dataset.sets
-        """
-
+        """Get the :obj:`Category` ID assigned to this dataset.sets"""
         return self.data['category_id']
 
     @property
     def category_name(self):
         """Name of the :obj:`Category` assigned to this dataset."""
-
         return self.data['category_name']
 
     @property
     def data_source(self):
         """Id of the data source of this dataset."""
-
         return self.data['data_source_id']
 
     @property
     def country(self):
-        """ISO 3166-1 alpha-3 code of the :obj:`Country`
-        of this dataset.
-        """
-
+        """ISO 3166-1 alpha-3 code of the :obj:`Country` of this dataset."""
         return self.data['country_id']
 
     @property
     def language(self):
-        """ISO 639-3 code of the language that corresponds to the data
-        of this dataset.
-        """
-
+        """ISO 639-3 code of the language that corresponds to the data of this dataset."""
```

Comment:
How about a link to the wikipedia source or better? https://en.wikipedia.org/wiki/ISO_639-3